src/peergos/shared/crypto/hash/Blake2b.java

```diff
@@ -0,0 +1,2092 @@
+package peergos.shared.crypto.hash;
+
+// From https://github.com/alphazero/Blake2b
+/* !!! Doost !!! */
```

Comment:
Doost?

Reply:
Not sure what it means, but it was in the original copyright message